packaging/suse/conf/registry.config.yml.in

```diff
@@ -0,0 +1,25 @@
+version: 0.1
+loglevel: info
+storage:
+  filesystem:
+    rootdirectory: /var/lib/docker-registry
```

Comment:
Inside of storage:

```yaml
delete:
  enabled: true
```
After all the trouble to upgrade to 2.1, it would be a shame to miss this feature ;)